src/base/utils/net.cpp

```diff
@@ -26,9 +26,12 @@
  * exception statement from your version.
  */
 
+#include "base/preferences.h"
```

Comment:
Now `isIPInAuthBypassSubnet` function isn't "utility" so just move it to AbstractWebApplication (e.g. `AbstractWebApplication::isClientInAuthBypassSubnet()`).
You can, of course, modify this function to turn it in "utility" again (ie to get rid from Preferences dependency), but in this case it would be better to have consistent interface (ie, both test address and subnets should have the same format - either QHostAddress or QString).